src/border_agent/border_agent.cpp

```diff
@@ -130,7 +132,7 @@ struct StateBitmap
         bitmap |= mAvailability << 5;
         bitmap |= mBbrIsActive << 7;
         bitmap |= mBbrIsPrimary << 8;
-
+        bitmap |= mEPSKcSupported << 11;
```

Comment:
This is okay but I am curios about using bit `11` here?  I think last one used is bit `8` for `mBbrIsPrimary`?

Reply:
Please ignore this. Found answer already. :)